Sources/RInAppMessaging/Extensions/UIFont+IAM.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit.UIFont
+
+/// Fonts referenced here must be pre-registered by the host application
+internal extension UIFont {
```

Comment:
Is it possible to unit test these by embedding a custom free use font?

Reply:
Added a glyph-less font file for tests but testing is tricky since fonts are globally registered and there's no way to unregister fonts at the same time (implementation not using UIFontDescriptors so we can fallback onto system fonts).